pt-BR/mail/chrome/messenger/focusPane.dtd

```diff
@@ -1,73 +1,73 @@
 <!-- strings used in both the focus pane and the full text search focus pane -->
-<!ENTITY focusPaneFilterByTags.label    "Favorite Topics">
-<!ENTITY focusPaneFilterByTagsMac.label "FAVORITE TOPICS">
-<!ENTITY focusPaneAllTags.label         "All">
-<!ENTITY focusPaneNoTags.label          "None">
+<!ENTITY focusPaneFilterByTags.label    "Tópicos favoritos">
+<!ENTITY focusPaneFilterByTagsMac.label "TÓPICOS FAVORITOS">
+<!ENTITY focusPaneAllTags.label         "Todos">
+<!ENTITY focusPaneNoTags.label          "Nenhum">
 
 <!-- strings used only in the full text search focus pane -->
-<!ENTITY focusPaneSearchScopeTitle.label       "Search">
-<!ENTITY focusPaneSearchScopeTitleMac.label    "SEARCH">
-<!ENTITY focusPaneSearchScopeEverywhere.label  "All Accounts">
-<!ENTITY focusPaneSearchAttribute.label        "Attributes">
-<!ENTITY focusPaneSearchAttributeMac.label     "ATTRIBUTES">
-<!ENTITY focusPaneHasAttachmentAttribute.label           "Has Attachment">
-<!ENTITY focusPaneUnreadAttribute.label           "Unread">
-<!ENTITY focusPaneFilterByDate.label    "Date">
-<!ENTITY focusPaneFilterByDateMac.label "DATE">
-<!ENTITY focusPaneAnyDate.label         "Any Date">
-<!ENTITY focusPaneToday.label           "Today">
-<!ENTITY focusPaneYesterday.label       "Yesterday">
-<!ENTITY focusPaneThisWeek.label        "This Week">
-<!ENTITY focusPaneThisMonth.label       "This Month">
-<!ENTITY focusPaneFilterByContact.label    "Favorite Contacts">
-<!ENTITY focusPaneFilterByContactMac.label "FAVORITE CONTACTS">
+<!ENTITY focusPaneSearchScopeTitle.label       "Pesquisar">
+<!ENTITY focusPaneSearchScopeTitleMac.label    "PESQUISAR">
+<!ENTITY focusPaneSearchScopeEverywhere.label  "Todas as contas">
+<!ENTITY focusPaneSearchAttribute.label        "Atributos">
+<!ENTITY focusPaneSearchAttributeMac.label     "ATRIBUTOS">
+<!ENTITY focusPaneHasAttachmentAttribute.label           "Tem anexo">
+<!ENTITY focusPaneUnreadAttribute.label           "Não-lido">
```

Comment:
@brunotiussi according to your previous translations, and to keep consistency, please chose either to use não-lido or não lido.